src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -71,6 +71,8 @@
 #define PASSIVE_PEER_KICK_TIME (60 * 1000000) // microseconds
 #define DROP_ON_SYNC_WEDGE_THRESHOLD (30 * 1000000000ull) // nanoseconds
 #define LAST_ACTIVITY_STALL_THRESHOLD (2.0f) // seconds
+#define DETECT_BROKEN_PEERS_TIME (90 * 1000000) // microseconds
+#define DROP_PEER_HEIGHT_INCREASE_THRESHOLD 128
```

Comment:
A bit concerned about low bandwidth peers. The restriction might be significantly relaxed (5x-10x times).

Reply:
I'll close this since it's too unsafe at this point before the fork, but... can you expand on this ?

Reply:
Given a peer with 1 Mbit/s bandwidth and Monero blocks 1MB in size, it will download just 11 blocks per 90 seconds.
It will be considered broken due to the current threshold, but actually it is not.

Reply:
OK. Closing now anyway.